deploy/sre-prometheus/extended-logging/101-parsed_elasticsearch_openshift-logging_elasticsearch-prometheus-rules.PrometheusRule.yaml

```diff
@@ -152,7 +152,19 @@ spec:
           for: 1h
           labels:
             namespace: openshift-logging
-            severity: critical
+            severity: warning
+        - alert: ElasticsearchDiskNeedsResizingSRE
+          annotations:
+            message: Cluster {{ $labels.cluster }} is predicted to be out of disk space within the next 6h and each of its disks should be resized to 400Gi.
```

Comment:
Is there a reason we need to include specific sizing guidance? Iirc, the SOP has some options for this?

Reply:
[Indeed](https://github.com/openshift/managed-cluster-config/pull/2547). From the SOP:
```
if currentStorage() < "1200Gi":
    notifyCustomer(action="Grow", target="1200Gi")
    growElasticStorageTo("1200Gi")
else:
    notifyCustomer(action="Reduce Logging")
    snoozeAlert("7 days")
```
Since the resize is always going to be to 1200Gi (spread across 3 400Gi disks), I figured why not specify it.